assets/keymaps/default-macos.json

```diff
@@ -301,6 +301,13 @@
       "cmd-ctrl-p": "editor::AddSelectionAbove",
       "cmd-alt-down": "editor::AddSelectionBelow",
       "cmd-ctrl-n": "editor::AddSelectionBelow",
+      "cmd-shift-k": "editor::DeleteLine",
+      "alt-up": "editor::MoveLineUp",
+      "alt-down": "editor::MoveLineDown",
+      "alt-shift-up": ["workspace::SendKeystrokes", "alt-shift-down up"],
```

Comment:
I can see why you had to do this, but I don't like using `SendKeystrokes` in this file. Could you add an optional field to the `DuplicateLine` action instead? It could be done similarly to the `advance_downwards` field of the `ToggleComments` action below. For compatibility, we'll need to annotate the field with `#[serde(default)]`, to make it optional in key binding files.